test/otel-collector-config.yaml

```diff
@@ -0,0 +1,21 @@
+receivers:
+  splunk_hec/logs:
```

Comment:
if we changed em_logs to em_events maybe it makes sense to change it here also?

Reply:
This name is internal to OTEL. It can take value of logs, metrics or traces. It has nothing to do with Splunk index names.